src/main/kotlin/com/terraformation/backend/config/TerrawareServerConfig.kt

```diff
@@ -106,6 +109,43 @@ class TerrawareServerConfig(
     /** Terraware support email config */
     val support: SupportConfig = SupportConfig(),
 ) {
+  class AtlassianConfig(
+      /** Atlassian account name */
+      val account: String? = null,
+
+      /** Atlassian host endpoint for Terraformation */
+      val apiHostname: String? = null,
+
+      /** Atlassian api key */
+      val apiToken: String? = null,
+
+      /** Enabled flag */
+      @DefaultValue("false") val enabled: Boolean = false,
+
+      /** Jira Bug Report type ID */
+      val bugReportTypeId: Int? = null,
+
+      /** Jira Feature Request type ID */
+      val featureRequestTypeId: Int? = null,
+
+      /** Service Desk ID */
+      val serviceDeskId: Int? = null,
```

Comment:
Could we look these IDs up at runtime (say, with a `by lazy` function so we only need to do it once per server process) rather than requiring them to be configured? Maybe have config options for the type _names_ with defaults of `"Bug Report"` and `"Feature Request"`.

Obviously configuring the IDs is also workable; I'm just thinking about how to reduce the number of setup steps and the amount of config we need for local dev and for CI.